Frontend/lib/screens/home/generate_address.dart

```diff
@@ -2,15 +2,15 @@ import 'package:flutter/material.dart';
 import 'package:frontend/utils/utils.dart';
 import 'package:frontend/widgets/widgets.dart';
 
-class KeyCreation extends StatefulWidget {
```

Comment:
can you explain "KeyCreation" class, just want to know what it does since we are not creating keys for the users.

Reply:
> can you explain "KeyCreation" class, just want to know what it does since we are not creating keys for the users.

It has been changed to generate_address